packages/platform-core/src/parameter_path_conversions.ts

```diff
@@ -11,7 +11,7 @@ export class ParameterPathConversions {
   /**
    * Convert a BackendIdentifier to a parameter prefix.
    */
-  static toParameterPrefix(backendId: BackendIdentifier | AppId): string {
+  toParameterPrefix(backendId: BackendIdentifier | AppId): string {
```

Comment:
I think these are fine to stay static. In this case we are really just using `ParameterPathConversions` as a namespace for a few functions. We can just ignore the lint rule for these instances (same comment applies to `BackendIdentifierFunctions`)

Reply:
I agree.
There are (very) rare cases where statics are fine. These two are good examples of that.
So opting out from rule for these is fine. And opting out from rule should trigger deeper discussion about whether the exception is justified or not.

Reply:
Done